WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/View Models/JetpackScanThreatViewModel.swift

```diff
@@ -309,8 +311,128 @@ struct JetpackScanThreatViewModel {
     }
 }
 
+extension JetpackScanThreatViewModel {
+
+    private static func attributedString(for context: JetpackThreatContext?) -> NSAttributedString? {
```

Comment:
How do you feel about extracting this into an extension on `JetpackThreatContext` that also takes a struct called something like `JetpackThreatContextRendererConfig`? Then the config struct can return the dictionaries for the NSAttributedString attributes?

Reply:
Extracted to a private `JetpackThreatContext` extension! ✨  de51d96

I decided to define a private extension inside `JetpackScanThreatViewModel` instead of WPKit, because attributed strings feel closer to a viewmodel than a data model.